pkg/scheduler/framework/preemption/preemption.go

```diff
@@ -274,8 +274,7 @@ func (ev *Evaluator) Preempt(ctx context.Context, state fwk.CycleState, pod *v1.
 			},
 		}
 		fitError.Diagnosis.NodeToStatus.SetAbsentNodesStatus(fwk.NewStatus(fwk.UnschedulableAndUnresolvable, "Preemption is not helpful for scheduling"))
-		// Specify nominatedNodeName to clear the pod's nominatedNodeName status, if applicable.
-		return framework.NewPostFilterResultWithNominatedNode(""), fwk.NewStatus(fwk.Unschedulable, fitError.Error())
+		return nil, fwk.NewStatus(fwk.Unschedulable, fitError.Error())
```

Comment:
After communicating with @sanposhiho in DM, we decided that this correction is necessary.